src/common/transformations/src/transformations/common_optimizations/transpose_sinking_split.cpp

```diff
@@ -171,7 +180,18 @@ ov::pass::TransposeSinkingSplitForward::TransposeSinkingSplitForward() {
 
         auto& main_node_output = pattern_to_output.at(main_node_label);
         auto main_node = main_node_output.get_node_shared_ptr();
-
+        auto split_node = as_type_ptr<Split>(main_node);
+        auto split_axis_constant = as_type_ptr<Constant>(split_node->input_value(1).get_node_shared_ptr());
```

Comment:
I think we should check that this is not `nullptr`?

Reply:
done